### PR TITLE
Make page titles consistent

### PR DESCRIPTION
**Pull request by danmitchell-**
#### Before
screen shot 2017-01-31 at 09 15 48
#### After
screen shot 2017-01-31 at 09 09 47

#### Before
screen shot 2017-01-30 at 16 50 27
#### After
screen shot 2017-01-30 at 16 57 05

#### Before
screen shot 2017-01-30 at 17 00 52
#### After
screen shot 2017-01-30 at 16 49 09

#### Before
screen shot 2017-01-30 at 16 50 22
#### After
screen shot 2017-01-30 at 16 49 04

#### Before
screen shot 2017-01-30 at 16 50 15
#### After
screen shot 2017-01-30 at 16 49 25

#### Before
screen shot 2017-01-30 at 16 50 19
#### After
screen shot 2017-01-30 at 16 57 43

#### Before
screen shot 2017-01-30 at 17 00 23
#### After
screen shot 2017-01-30 at 16 58 13

### CODE REVIEW

**Comment by michaelabenyohai** [comment]
@danmitchell- you have a failing test unfortunately!

Also, as discussed, we need to keep the "View this register" link (which could do with being renamed).

**Comment by danmitchell-** [comment]
All passing now @michaelabenyohai and I added a screenshot of the register register record view :)

File: src/main/resources/templates/item.html [hunk 1]
```diff
@@ -20,15 +20,14 @@
 
       <div class="grid-row">
         <div class="column-two-thirds">
-          <p></p>
+          <h1 class="heading-large">Item</h1>
```
Comment (tvararu) [inline src/main/resources/templates/item.html]: This isn't very descriptive for assistive technologies but I don't have a better idea about what to say instead, so let's leave it like this for now.

(Lots of screen readers allow you to press `1` to go to the H1 on the current page, and when read out, "Item" isn't very useful)

**Comment by tvararu** [comment]
@michaelabenyohai can you merge this if it's all cool?

**Pull request merged**